core/config/default-config.js

```diff
@@ -610,6 +603,8 @@ const defaultConfig = {
         {id: 'errors-in-console', weight: 1, group: 'best-practices-general'},
         {id: 'valid-source-maps', weight: 0, group: 'best-practices-general'},
         {id: 'inspector-issues', weight: 1, group: 'best-practices-general'},
+        {id: 'themed-omnibox', weight: 1, group: 'best-practices-general'},
+        {id: 'viewport', weight: 2, group: 'best-practices-general'},
```

Comment:
I don't think we need this audit in performance, seo *and* best practices. I think we should remove it from SEO if we really want it here, but that's a different conversation.